label_buddy/static/js/annotation_page.js

```diff
@@ -208,23 +208,36 @@ function fixNumberOfRegions() {
         $(btn).find('#count').text(counter++ + ".")
     }
 }
-//----------------------------------------------------------------------------------------------
 
+function get_loaded_precentage(audio) {
+    var buffered = audio.buffered; // returns the buffered portion of the audio
+    var loaded; // the loaded portion of the audio
+    var played; // the played portion of the audio
+
+    if (buffered.length) {
```

Comment:
why do we need this?

Reply:
Previous implementation caused an error if the buffer was empty.
Changed it.